packages/graphics/src/Graphics.ts

```diff
@@ -1151,9 +1151,18 @@ export class Graphics extends Container
             // but may be more than one plugins for graphics
             if (!DEFAULT_SHADERS[pluginName])
             {
-                const sampleValues = new Int32Array(16);
+                const batchRenderer = renderer.plugins[this.pluginName];
+                let MAX_TEXTURES = 16;
 
-                for (let i = 0; i < 16; i++)
+                if (batchRenderer && typeof batchRenderer.MAX_TEXTURES === 'number'
```

Comment:
Question for @ivanpopelyshev or @SukantPal 
Since MAX_TEXTURES exists on AbstractBatchRenderer, can't we assume that MAX_TEXTURES property always exists?
Just wondering if we need this defensive if statement each time

Reply:
@dumganhar To get the MAX_TEXTURES, please use hardcore the batch plugin (I.e. use renderer.plugins.batch) because the user may be using a custom object renderer that does not provide this property.

@themoonrat If referring plugins.batch, you’re assumption is correct. But not all object renderers need be batch renderer (or extend AbstractBatchRenderer)

Reply:
@themoonrat @SukantPal , Pull Request was updated, please review again.